### PR TITLE
renamed materialize-views sub command

### PR DESCRIPTION
**Pull request by de-code**
as a consequence of https://github.com/elifesciences/data-pipeline-bigquery-views/pull/47

renamed `materialize` to `materialize-views`

### CODE REVIEW

**Review by giorgiosironi — APPROVED** [review]

**Comment by giorgiosironi** [comment]
I assume there is backward compatibility in the code i.e. the old command is still working.

**Comment by de-code** [comment]
No, since it's not a published library yet there doesn't seem to be much need to make it backward compatible.

**Comment by giorgiosironi** [comment]
I mean in general, when the contract between the code and the formula changes usually the code first deprecates the old solution and only removes it once the formula has moved off from it.

**Comment by de-code** [comment]
> I mean in general, when the contract between the code and the formula changes usually the code first deprecates the old solution and only removes it once the formula has moved off from it.

Yes, true. I thought we can get away without it in this case as the function is barely working now and only ran once a day. So if we merge this PR first after 10am, then we still have some time to merge the other PR after it.

**Pull request merged**
